CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ## Changelog
 
+### 4.0.1 (Next)
+
+* [#215](https://github.com/alexa-js/alexa-app/pull/215): Fix Syntax of Session Details to match with Docs - [@danielstieber](https://github.com/danielstieber).
```

Comment:
Minor: lowercase everything except `Fix`, and make it `Fixed`. Also maybe be clearer about what session details we're talking about. I suggest: `Fixed session.details ...bla bla bla`.